src/m365/spo/commands/contenttype/contenttype-set.ts

```diff
@@ -19,6 +21,7 @@ interface Options extends GlobalOptions {
   listTitle?: string;
   listId: string;
   listUrl: string;
+  updateChildren: boolean;
```

Comment:
if this is optional should this be like:
```suggestion
  updateChildren?: boolean;
```

Reply:
Hi @Adam-it,

All our flags are written as not-Nullable. That's because the value is either true or it's not. 

Minimist will parse it as `undefined` if the flag is unused. So strictly speaking you are right. But for flags there's no difference between `false` and `undefined`. 

There's a point of discussion here: We could update our code and take care that flags parse as `false` if unused. But it won't add anything new, so I don't think it's a useful way to spend our time :)

Reply:
understood. Thanks for the explanation 👍 lets leave it